frontend/src/pulse/components/PulseEditChannels.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component, PropTypes } from "react";
 
 import _ from "underscore";
+import i from "icepick";
```

Comment:
question.

If I have a cool loop with `i` as my index variable I will inadvertently end up shadowing the icepick `i` variable. Then if I try to do something like `i.assocIn` I'll get a `assocIn is not a function`, breaking all of the Metabase.

I would posit that reserving a common variable name like `i` is bound to lead to just such chaos. Why would the library writers do such a thing to us? Is there any way we can import it as an alias?